From: Dmitry Shachnev <mitya57@debian.org>
Date: Fri, 29 Jun 2018 14:54:26 +0300
Subject: Use only TTF and WOFF2 fonts for Lato, only WOFF2 for RobotoSlab

Also use different file names for compatibility with packages built
against sphinx-rtd-theme 0.2.4.
---
 src/sass/_theme_font_local.sass | 15 ++++++---------
 1 file changed, 6 insertions(+), 9 deletions(-)

diff --git a/src/sass/_theme_font_local.sass b/src/sass/_theme_font_local.sass
index 48f22e7..31da232 100644
--- a/src/sass/_theme_font_local.sass
+++ b/src/sass/_theme_font_local.sass
@@ -1,30 +1,27 @@
-$lato-font-path: "~lato-font/fonts"
-$roboto-font-path: "~roboto-fontface/fonts/roboto-slab"
-
 @font-face
   font-family: 'Lato'
-  src: url('#{$lato-font-path}/lato-normal/lato-normal.woff2') format('woff2'), url('#{$lato-font-path}/lato-normal/lato-normal.woff') format('woff')
+  src: url('../fonts/Lato-Regular.woff2') format('woff2'), url('../fonts/Lato-Regular.ttf') format('truetype')
   font-weight: 400
   font-style: normal
   font-display: $font-display
 
 @font-face
   font-family: 'Lato'
-  src: url('#{$lato-font-path}/lato-bold/lato-bold.woff2') format('woff2'), url('#{$lato-font-path}/lato-bold/lato-bold.woff') format('woff')
+  src: url('../fonts/Lato-Bold.woff2') format('woff2'), url('../fonts/Lato-Bold.ttf') format('truetype')
   font-weight: 700
   font-style: normal
   font-display: $font-display
 
 @font-face
   font-family: 'Lato'
-  src: url('#{$lato-font-path}/lato-bold-italic/lato-bold-italic.woff2') format('woff2'), url('#{$lato-font-path}/lato-bold-italic/lato-bold-italic.woff') format('woff')
+  src: url('../fonts/Lato-BoldItalic.woff2') format('woff2'), url('../fonts/Lato-BoldItalic.ttf') format('truetype')
   font-weight: 700
   font-style: italic
   font-display: $font-display
 
 @font-face
   font-family: 'Lato'
-  src: url('#{$lato-font-path}/lato-normal-italic/lato-normal-italic.woff2') format('woff2'), url('#{$lato-font-path}/lato-normal-italic/lato-normal-italic.woff') format('woff')
+  src: url('../fonts/Lato-Italic.woff2') format('woff2'), url('../fonts/Lato-Italic.ttf') format('truetype')
   font-weight: 400
   font-style: italic
   font-display: $font-display
@@ -33,12 +30,12 @@ $roboto-font-path: "~roboto-fontface/fonts/roboto-slab"
   font-family: 'Roboto Slab'
   font-style: normal
   font-weight: 400
-  src: url('#{$roboto-font-path}/Roboto-Slab-Regular.woff2') format('woff2'), url('#{$roboto-font-path}/Roboto-Slab-Regular.woff') format('woff')
+  src: url('../fonts/RobotoSlab-Regular.woff2') format('woff2')
   font-display: $font-display
 
 @font-face
   font-family: 'Roboto Slab'
   font-style: normal
   font-weight: 700
-  src: url('#{$roboto-font-path}/Roboto-Slab-Bold.woff2') format('woff2'), url('#{$roboto-font-path}/Roboto-Slab-Bold.woff') format('woff')
+  src: url('../fonts/RobotoSlab-Bold.woff2') format('woff2')
   font-display: $font-display
